op-e2e/sequencer_failover_setup.go

```diff
@@ -54,56 +53,66 @@ func setupSequencerFailoverTest(t *testing.T) (*System, map[string]*conductor) {
-		{sequencer3Name, false},
+		{Sequencer1Name, conductorRpcPorts[Sequencer1Name], true}, // one in bootstrap mode so that we can form a cluster.
+		{Sequencer2Name, conductorRpcPorts[Sequencer2Name], false},
+		{Sequencer3Name, conductorRpcPorts[Sequencer3Name], false},
 	}
 	for _, cfg := range conductorCfgs {
 		cfg := cfg
 		nodePRC := sys.RollupNodes[cfg.name].HTTPEndpoint()
 		engineRPC := sys.EthInstances[cfg.name].HTTPEndpoint()
-		conductors[cfg.name] = setupConductor(t, cfg.name, t.TempDir(), nodePRC, engineRPC, cfg.bootstrap, *sys.RollupConfig)
+		conductors[cfg.name] = setupConductor(t, cfg.name, t.TempDir(), nodePRC, engineRPC, cfg.bootstrap, *sys.RollupConfig, cfg.port)
 	}
 
+	// start sequencer 1
+	unsafeHead, err := sys.Clients[Sequencer1Name].BlockByNumber(ctx, nil)
+	require.NoError(t, err)
+	require.NoError(t, sys.RollupClient(Sequencer1Name).StartSequencer(ctx, unsafeHead.Hash()))
+
+	// 1 batcher starts in started mode and listens to all 3 sequencers.
+	setupBatcher(t, sys)
+
 	// form a cluster
-	c1 := conductors[sequencer1Name]
-	c2 := conductors[sequencer2Name]
-	c3 := conductors[sequencer3Name]
+	c1 := conductors[Sequencer1Name]
+	c2 := conductors[Sequencer2Name]
+	c3 := conductors[Sequencer3Name]
 
 	require.NoError(t, waitForLeadershipChange(t, c1, true))
-	require.NoError(t, c1.client.AddServerAsVoter(ctx, sequencer2Name, c2.ConsensusEndpoint()))
-	require.NoError(t, c1.client.AddServerAsVoter(ctx, sequencer3Name, c3.ConsensusEndpoint()))
+	require.NoError(t, c1.client.AddServerAsVoter(ctx, Sequencer2Name, c2.ConsensusEndpoint()))
+	require.NoError(t, c1.client.AddServerAsVoter(ctx, Sequencer3Name, c3.ConsensusEndpoint()))
 	require.True(t, leader(t, ctx, c1))
 	require.False(t, leader(t, ctx, c2))
 	require.False(t, leader(t, ctx, c3))
 
-	// weirdly, batcher does not submit a batch until unsafe block 9.
-	// It became normal after that and submits a batch every L1 block (2s) per configuration.
-	// Since our health monitor checks on safe head progression, wait for batcher to become normal before proceeding.
-	require.NoError(t, wait.ForNextSafeBlock(ctx, sys.Clients[sequencer1Name]))
-	require.NoError(t, wait.ForNextSafeBlock(ctx, sys.Clients[sequencer1Name]))
-	require.NoError(t, wait.ForNextSafeBlock(ctx, sys.Clients[sequencer1Name]))
-
-	// make sure conductor reports all sequencers as healthy, this means they're syncing correctly.
-	require.True(t, healthy(t, ctx, c1))
-	require.True(t, healthy(t, ctx, c2))
-	require.True(t, healthy(t, ctx, c3))
+	require.Eventually(t, func() bool {
+		return healthy(t, ctx, c1)
+	}, 30*time.Second, 500*time.Millisecond, "Expected sequencer 1 to become healthy")
+	require.Eventually(t, func() bool {
+		return healthy(t, ctx, c2)
+	}, 30*time.Second, 500*time.Millisecond, "Expected sequencer 2 to become healthy")
+	require.Eventually(t, func() bool {
+		return healthy(t, ctx, c3)
+	}, 30*time.Second, 500*time.Millisecond, "Expected sequencer 3 to become healthy")
```

Comment:
You might benefit from putting these all inside a single `Eventually` Clause, seeing as all three hosts are progressing toward health at the same time.
```suggestion
	require.Eventually(t, func() bool {
		return healthy(t, ctx, c1) &&
		healthy(t, ctx, c2) &&
		healthy(t, ctx, c3)
	}, 30*time.Second, 500*time.Millisecond, "Expected sequencer 1 to become healthy")
```

Reply:
Done